lib/song.rb

```diff
@@ -0,0 +1,32 @@
+require 'pry'
+
+class Song
+  attr_accessor :name, :artist, :genre
+  @@count = 0
+  @@artists = []
+  @@genres = []
+  def initialize(name, artist, genre)
+    @name = name
+    @artist = artist
+    @genre = genre
+    
+    @@count += 1
+    @@artists << artist
+    @@genres << genre
+  end
+  def self.count
+    @@count 
+  end
+  def self.artists
+    @@artists.uniq
+  end
+  def self.genres
+    @@genres.uniq
+  end
+  def self.genre_count
+    @@genres.tally
+  end
+  def self.artist_count
+    @@artists.tally
+  end
```

Comment:
Try putting lines between your method definitions. 
```suggestion
  def self.count
    @@count 
  end
  
  def self.artists
    @@artists.uniq
  end
  
  def self.genres
    @@genres.uniq
  end
  
  def self.genre_count
    @@genres.tally
  end
  
  def self.artist_count
    @@artists.tally
  end
```